yarn.lock

```diff
@@ -12027,13 +12027,20 @@ sass-lookup@^3.0.0:
   dependencies:
     commander "^2.16.0"
 
-sass@^1.26.3, sass@^1.34.1:
+sass@^1.26.3:
```

Comment:
Nit: we could dedupe this if you remove this entry in the lock file and then run `yarn` again.